app/uk/gov/nationalarchives/omega/editorial/views/editSet.scala.html

```diff
@@ -97,8 +113,17 @@
               )
           )
       ),
-      caption = Some(heading) ,
+      caption = Some(
+        messages(
+          "edit-set.table-caption",
+          page.numberOfFirstEntry.toString,
+          page.numberOfLastEntry.toString,
+          page.totalNumberOfEntries.toString
+        )
+      ),
       captionClasses = "govuk-table__caption--m",
       firstCellIsHeader = true
   ))
+
+  @govukPagination(page.pagination)
```

Comment:
Done, though what I would really like is to have the pagination template in the HMRC library use messages strings instead of hardcoded. I looked at opening a PR but I think that would be too much work.

Reply:
I'm not 100% happy with the updated solution by the way, if you think there's a better way I'd be open to suggestions.

Reply:
It's a decent solution.

It's a curious lack in the lib, but no - we shouldn't wait for a PR to be accepted.

Confirmed.